Test/TestGeometry/tests/options/geant_overlap_test_fcchh.py

```diff
@@ -10,7 +10,7 @@
                         detector='SimG4DD4hepDetector',
                         physicslist="SimG4FtfpBert",
                         actions="SimG4FullSimActions",
-                        g4PostInitCommands=['/geometry/test/recursion_depth 3',  '/geometry/test/run'])
+                        g4PostInitCommands=['/geometry/test/recursion_depth 2',  '/geometry/test/run'])
```

Comment:
why this change? time?

Reply:
yes, this test was taking more than 20 min like this. I can change it back, though!

Reply:
But did you check it is useful with this depth? Maybe we could run it only in nightlies or sth? @JavierCVilla , what do you think?